tests/Shared/Playwright/Playwright.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)*.cs" Link="Playwright" />
+
+    <PackageReference Include="Microsoft.Playwright" />
+  </ItemGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)provisioning.targets" />
```

Comment:
Is there a reason this is a separate .targets file? Why not just put this in Playwright.targets itself?